examples/async_pcb_insert_drq/async_drq_randomized.py

```diff
@@ -92,31 +92,26 @@ def print_green(x):
 
 
 SHOULD_PAUSE = False  # flag to pause actor/learner agents
+PAUSE_LOCK = threading.Lock()
```

Comment:
Oh srry, i meant using [`threading Event`](https://docs.python.org/3/library/threading.html#event-objects), which is easier than having 2 variables. Also, can rename it to `PAUSE_EVENT_FLAG`. Personally i dont like using global, but i think this should be okay since it mainly serves as an example. Thanks

Reply:
I replaced it with a threading Event. Please feel free to test it out.

@global: I also do not prefer global, and would usually store these as class variables.